lib/api/etas_api.rb

```diff
@@ -0,0 +1,59 @@
+require "cluster"
+require "clusterable/ht_item"
+require "services"
+require "sinatra"
+
+class EtasAPI < Sinatra::Base
+  set :show_exceptions, ENV["show_sinatra_exceptions"] || false
+  no_matching_data_error = {"application_error" => "no matching data"}.to_json
+
+  get "/ping" do
+    "pong"
+  end
+
+  get "/etas/item_access" do
```

Comment:
I would recommend versioning the API right from the get-go -- in other words `get "/v1/path/to/a/thing"` etc. And also echo Aaron's comment about other routes. We should try to get a spec for a basic suite of endpoints that if designed well should protect us from too much version churn.

Reply:
rights_api has an example of redirecting to the versioned URL from an unversioned one.

Reply:
Yes. Implemented!